src/Behat/FlexibleMink/Context/FlexibleContext.php

```
@@ -151,6 +151,34 @@ public function assertFieldNotExists($fieldName)
     /**
      * {@inheritdoc}
      *
+     * @Then I should see the following lines in order:
+     */
+    public function assertLinesInOrder(TableNode $table)
+    {
+        $session = $this->getSession();
+        $page = $session->getPage()->getText();
+
+        $lines = $table->getColumn(0);
```

Maybe throw an exception if they were to add a table that has more than one column?